packages/docusaurus-preset-bootstrap/src/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const path = require('path');
+
+module.exports = function () {
```

Comment:
Why we don't use the es6 syntax? `export function() {}` I noticed that this is common when we have plugin, themes and preset export definitions

Reply:
This is on Node and I believe it doesn't support ES6 exports syntax.